CMakeLists.txt

```diff
@@ -62,10 +62,10 @@ option(XEUS_DOWNLOAD_GTEST "build gtest from downloaded sources" OFF)
 
 if (XEUS_STATIC_DEPENDENCIES)
     set(CPPZMQ_TARGET_NAME cppzmq-static)
-    set(OPENSSL_USE_STATIC_LIBS ON)
+    set(OPENSSL_USE_STATIC_LIBS ON CACHE)
 else()
     set(CPPZMQ_TARGET_NAME cppzmq)
-    set(OPENSSL_USE_STATIC_LIBS OFF)
```

Comment:
With this change, we do not override values passed at the command line for `OPENSSL_USE_STATIC_LIBS`.